consensus/validator.go

```diff
@@ -89,6 +89,41 @@ func (consensus *Consensus) prepare() {
 	consensus.switchPhase("Announce", FBFTPrepare)
 }
 
+// sendCommitMessages send out commit messages to leader
+func (consensus *Consensus) sendCommitMessages(blockObj *types.Block) {
+	// Sign commit signature on the received block
+	commitPayload := signature.ConstructCommitPayload(consensus.ChainReader,
+		blockObj.Epoch(), blockObj.Hash(), blockObj.NumberU64(), blockObj.Header().ViewID().Uint64())
+	groupID := []nodeconfig.GroupID{
+		nodeconfig.NewGroupIDByShardID(nodeconfig.ShardID(consensus.ShardID)),
+	}
+	for _, key := range consensus.priKey {
+		if !consensus.IsValidatorInCommittee(key.Pub.Bytes) {
+			continue
+		}
+
+		networkMessage, _ := consensus.construct(
```

Comment:
shouldn't ignore the err. If err != nil, networkMessage will be nil.

Reply:
will check.